### ISSUE TITLE
PushNotification.invokeApp does not work in Android 12+ , possibly because of Notification trampoline restrictions

### ISSUE THREAD

**Issue by sujayjaju** [zo0r/react-native-push-notification#2334]
# Bug

## Environment info

`react-native info` output:

```bash
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 1.86 GB / 15.84 GB
  Binaries:
    Node: 14.20.0 - ~\AppData\Local\Temp\yarn--1670940874250-0.047929254082552886\node.CMD
    Yarn: 1.22.19 - ~\AppData\Local\Temp\yarn--1670940874250-0.047929254082552886\yarn.CMD
    npm: 7.22.0 - ~\Dev\Ancoria\PFMApp\app\node_modules\.bin\npm.CMD
    Watchman: 20220911.220011.0 - C:\Lib\watchman-v2022.09.12.00-windows\bin\watchman.EXE
  SDKs:
    Android SDK: Not Found
    Windows SDK: Not Found
  IDEs:
    Android Studio: AI-212.5712.43.2112.8815526
    Visual Studio: Not Found
  Languages:
    Java: 11.0.16.1 - C:\Program Files\Common Files\Oracle\Java\javapath\javac.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: ^0.67.0-0 => 0.67.4
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

```

Library version: 8.1.1

## Steps To Reproduce

1. Setup up a local notification with actions and invokeApp false
```javascript
PushNotification.localNotification({
...
    invokeApp: false, 
    actions,
...
})
```
2. Call invoke app in onAction:
```javascript
    onAction: function (notification) {
      // process
      PushNotification.invokeApp(notification);
    },
```
...

Describe what you expected to happen:

**1. App should come to foreground, but it does not**

## Reproducible sample code
See above

**Comment by ashishmangukiya** [comment]
I am facing the same issue.   Please let me know if you get any solution.

**Comment by sujayjaju** [comment]
@ashishmangukiya I am in the process to switch to notifee

**Comment by karakellef** [comment]
Any solutions?

**Comment by fcaldarelli** [comment]
Fixed with #2352
